drivers/adc/adc_mcux_lpadc.c

```diff
@@ -552,6 +570,7 @@ static const struct adc_driver_api mcux_lpadc_driver_api = {
 		.irq_config_func = mcux_lpadc_config_func_##n,				\
 		.pincfg = PINCTRL_DT_INST_DEV_CONFIG_GET(n),			\
 		.ref_supplies = mcux_lpadc_ref_supplies_##n, \
+		.ref_voltage_val = DT_INST_PROP_OR(n, nxp_reference_supply_value, 0), \
```

Comment:
See comment on DT binding- I think we should extract this value from `zephyr,vref-mv`